scripts/obtain/install.sh

```diff
@@ -123,196 +130,466 @@ check_pre_reqs() {
         [ -z "$($LDCONFIG_COMMAND -p | grep gettext)" ] && say_err "Unable to locate gettext. Install gettext to continue" && _failing=true
     fi
 
-    if [ "$_failing" = true ]; then
+    if [ "$failing" = true ]; then
```

Comment:
good point, above should be failing, not sure why to use underscore - merge conflict must have broken it